LICENSE.TXT

```diff
@@ -102,7 +102,8 @@ RapidWright (Apache 2.0)
   - JetBrains Java Annotations 20.1.0 (Apache 2.0)
   - jzlib 1.1.3 (BSD-style License)
   - Apache Commons IO 2.11.0 (Apache 2.0)
-
+  - Zstd-jni 1.5.5-1 (2-clause BSD)
+      - Zstandard 1.5.5 (3-clause BSD or GPLv2)

```

Comment:
Spacing still wonky here?

Reply:
Not sure what you mean.  Zstd-jni depends on Zstandard, this is just an indent (see examples above).

Reply:
Oooh, I see! I didn't notice the indent out of my view. Sorry!